packages/atomic/src/pages/examples/instantResults.html

```diff
@@ -29,7 +29,7 @@
       <atomic-search-layout mobile-breakpoint="700px">
         <atomic-layout-section section="search">
           <atomic-search-box suggestion-timeout="1000">
-            <atomic-search-box-recent-queries icon="assets://xml.svg"> </atomic-search-box-recent-queries>
+            <!-- <atomic-search-box-recent-queries icon="assets://xml.svg"> </atomic-search-box-recent-queries> -->
```

Comment:
Did you mean to commit this change?
```suggestion
            <atomic-search-box-recent-queries icon="assets://xml.svg"> </atomic-search-box-recent-queries>
```